test/export/StructureDefinitionExporter.test.ts

```diff
@@ -1,8 +1,14 @@
 import { StructureDefinitionExporter } from '../../src/export';
```

Comment:
For this file in general, in my usual line of inconsequential typo feedback, there are a few instances of `FixValue` or `FixValueRule` that should be changed to `FixedValue` or `FixedValueRule`, respectively.

Reply:
Typos should be addressed now! Good catch!